packages/gestalt/src/FormElement.css

```diff
@@ -36,3 +49,14 @@
   composes: gray from "./Colors.css";
   composes: lightGrayBg from "./Colors.css";
 }
+
```

Comment:
@AlbertCarreras can you ensure you have `stylelint` installed as a vscode extension? Looks like we have double line breaks here